core/unit_test/cuda/TestCuda_ReducerViewSizeLimit.cpp

```diff
@@ -86,6 +86,13 @@ struct MDArrayReduceFunctor {
   KOKKOS_INLINE_FUNCTION void final(value_type) const {}
 };
 
+template <class PolicyType, int S>
+struct FunctorFor {
+  double static_array[S];
```

Comment:
```suggestion
template <class PolicyType, std::size_t Bytes>
struct FunctorFor {
  char a[Bytes];
```
"static" is not a good name here because one could think it is a static data member.

Also please add a comment why the array is needed.

Reply:
I removed it since the test doesn't need it
👍 118bdf5a4171920213a98204d47f92c62d6ff2d7